docs/source/install/index.rst

```diff
@@ -60,13 +60,13 @@ For Fedora/RHEL/CentOS users, run:
 
    sudo yum install -y gcc-c++ python3-devel make cmake
 
-Create conda environment for development, run:
+Create conda environment for development. To see the usage of create script, run:
```

Comment:
how about listing the detailed command of creating env for cpu build and gpu build. then remind the user to run `-h` for more usage?